README.md

```diff
@@ -1,110 +1,31 @@
 ![](https://raw.githubusercontent.com/auth0/wp-auth0/master/banner-1544x500.png)
 
-Wordpress Plugin for Auth0
+Login by Auth0
 ====
 
 Single Sign On for Enterprises + Social Login + User/Passwords. For all your WordPress instances. Powered by Auth0.
 
-Demo: <http://auth0wp.azurewebsites.net>
-
-Documentation: <https://auth0.com/docs/cms>
+Download and install <https://wordpress.org/plugins/auth0/>
+Documentation: <https://auth0.com/docs/cms/wordpress>
 
 ## Important note on 3.5
 
 This is a major update that requires changes to your Auth0 Dashboard to be completed. You can save a new [API token](https://auth0.com/docs/api/management/v2/tokens#get-a-token-manually) in your Basic settings in wp-admin before upgrading and the changes will be made automatically during the update. Otherwise, please review your [Client Advanced Settings](https://auth0.com/docs/cms/wordpress/configuration#client-setup), specifically your Grant Types, and [authorize your Client for the Management API](https://auth0.com/docs/cms/wordpress/configuration#authorize-the-client-for-the-management-api). 
 
-## Contributions
-
-All PR should be done towards the `dev` branch.
-
 ## Installation
```

Comment:
I think there is no harm in duplicating core information like install, if you discover the repo *somehow* then the README should be enough to get started without having to go off elsewhere. The other repos are like this.

Reply:
@cocojoe - The other repos like SDKs? The problem comparing those is that the repo is the starting point for those whereas wp.org/plugins is where most people will start with this plugin. All of the "how to use this plugin" stuff should be on docs, where we have much better control over presentation/content. All of the basic stuff and "why to use this" should be in the WP.org readme. The repo readme is probably best as a list of links, for the most part.

What I really want to avoid, though, is maintain the same steps/info in 2 or more different places.